docs/platform/what-is-tizen/profiles/iot.md

```diff
@@ -13,28 +13,30 @@ Tizen IoT supports the following environment for development:
 
 ## Develop applications for Tizen IoT
 To develop applications for the IoT profile:
-- [Create Your First Tizen IoT .NET Application](../../../application/dotnet/get-started/iot/first-app.md)
+- [Create Your First Tizen IoT .NET Application](../../../application/dotnet/get-started/first-app.md)
+- [Create Your First Tizen IoT Native Service Application](../../../application/native/get-started/iot/first-app.md)
 
 ## Create Tizen IoT images
-Tizen IoT provides a building block pool of components mainly based on the [Tizen native API sets](../../../application/native/api/wearable/latest/index.html).
+Tizen IoT provides a building block pool of components mainly based on the [Tizen native API sets](../../../application/native/api/iot-headed/latest/index.html).
```

Comment:
API names should be capitalized. Ref: https://github.com/Samsung/tizen-docs/blob/master/styleguide/naming-rules.md
```suggestion
Tizen IoT provides a building block pool of components mainly based on the [Tizen Native API sets](../../../application/native/api/iot-headed/latest/index.html).
```